feat: Display behavior logs (#2531)
webrecorder/browsertrix: frontend/docs/docs/user-guide/archived-items.md

To combine multiple archived items and download them all as a single WACZ file, add them to a collection and [download the collection](collection.md#download-a-collection).

### Logs

> View a list of errors and behavior logs that may have been encountered during crawling. Clicking a log in the list will reveal additional information.

Reviewer: ```suggestion
View a list of errors and behavior logs that were generated during crawling. Clicking a log entry in the list will reveal additional information.
```

> All log entries with that were recorded in the creation of the archived item can be downloaded in JSONL format by pressing the _Download All Logs_ button.

Reviewer: ```suggestion
Only a subset of the logs generated by the crawler are visible in Browsertrix. All log entries that were recorded in the creation of the archived item can be downloaded in JSONL format by pressing the _Download All Logs_ button.
```
